packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -75,19 +81,31 @@ export class Context implements FlexTreeContext, IDisposable {
 	private disposed = false;
 
 	/**
-	 * @param forest - the Forest
-	 * @param editor - an editor that makes changes to the forest.
+	 * The forest from the {@link checkout}.
+	 */
+	public readonly forest: IForestSubscription;
```

Comment:
What's a forest supposed to be? I tried reading into existing code but still a bit confused

Reply:
A forest is a collection of trees! :)

It is where SharedTree stores the in-memory content of the tree. If you were to ask, "where is the actual data of a SharedTree?" - the answer is that it's in the forest. 

It's a "forest" (multiple trees) rather than merely a single tree, because SharedTree has multiple trees under the hood used for various purposes. Though, there is one particular tree in every forest that is the "special" tree, the main tree, the thing that end-users interact with via the SharedTree APIs.

Forests are read via "cursors" - for example, you might allocate a cursor, move it to the top of one of the trees in the forest, and then walk down the tree via the cursor, reading or writing things as you go.

Multiple checkouts can exist for a single SharedTree, and each checkout has its own forest. This is because checkouts might be in different states (e.g. viewing the tree at different points in time, or viewing a temporary branch of the tree's state rather than the "main" branch state), and so each one needs a forest to hold that state.

Reply:
https://github.com/microsoft/FluidFramework/tree/main/packages/dds/tree/src/core/forest

I think this could be a help for you.